orienteer-users/src/main/java/org/orienteer/users/hook/OrienteerUserRoleHook.java

```diff
@@ -0,0 +1,34 @@
+package org.orienteer.users.hook;
+
+import com.orientechnologies.orient.core.db.document.ODatabaseDocument;
+import com.orientechnologies.orient.core.hook.ODocumentHookAbstract;
+import com.orientechnologies.orient.core.metadata.security.ORestrictedOperation;
+import com.orientechnologies.orient.core.metadata.security.ORole;
+import com.orientechnologies.orient.core.record.impl.ODocument;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class OrienteerUserRoleHook extends ODocumentHookAbstract {
```

Comment:
No JavaDoc - it leads to codestyle check fail

Reply:
Fixed in 956cc73